docroot/themes/custom/uids_base/templates/views/views-view-unformatted--list-card.html.twig

```diff
@@ -42,6 +43,12 @@
     {% endif %}
 
   {% endfor %}
+
+  {% if '1' in row.content['#node'].fields.field_person_website_link_direct.value  %}
+    {% set node_link = view.style_plugin.getField(key, 'field_person_website')|striptags|trim %}
+  {% else %}
+    {% set node_link = view.style_plugin.getField(key, 'view_node')|striptags|trim %}
+  {% endif %}
```

Comment:
Same note as above. Also wondering about why `|striptags|trim` is being used here?

Reply:
I thought in midsprint demo we decided to keep it in the template. Partially due to concerns the template was being used by more than just sitenow people, and partially because of the UIDS card work. There was something about the card work involving refactoring anyways, so this would be temporary and could be kept as simple as possible. 

`|striptags|trim` was added because at one point the link wouldn't work due to html tags being added. I will double-check if this is still happening/still necessary.

Reply:
> I thought in midsprint demo we decided to keep it in the template.

Thanks for reminding me of this. Yes, that makes sense and let's keep it this way for now. Maybe add a comment to the effect that it intended to be temporary so that it is clear for the next person looking at it that they should not use it as an example of how to do things.

> `|striptags|trim` was added because at one point the link wouldn't work due to html tags being added. I will double-check if this is still happening/still necessary.

Okay, I see now. Thanks for the explanation.

Reply:
Good news! 1. I remembered to go back and check this, and 2. `|striptags|trim` is no longer needed, it works as it should without it.

Reply:
Wait I was wrong I looked at the /people view instead of the People listblock. It is still needed for the listblock to work.

Reply:
Can we leave striptags|trim in until we address them in https://github.com/uiowa/uiowa/issues/6030? If so, based on https://github.com/uiowa/uiowa/pull/6053#discussion_r1064710950 can this also be resolved?